src/styles/Components/src/tokens-list.css

```diff
@@ -0,0 +1,40 @@
+:root {
+  --denhaag-list-font-family: var(--denhaag-font-family);
+
+  /* ListItem */
+  --denhaag-listitem-padding-inline: 16px;
+  --denhaag-listitem-padding-block: 12px;
+  --denhaag-listitem-border-size: 2px;
+  --denhaag-listitem-focus-border-color: var(--denhaag-ocher-4);
+  --denhaag-listitem-focus-border-width: var(--denhaag-focus-border-width);
+  --denhaag-listitem-focus-border-style: var(--denhaag-focus-border-style);
+  --denhaag-listitem-padding-block-secondary: 8px;
+  --denhaag-listitem-active-border-color: var(--denhaag-blue-3);
+
+  /* ListItemIcon */
+  --denhaag-listitem-icon-color: var(--denhaag-grey-4);
+  --denhaag-listitem-icon-focus-color: var(--denhaag-blue-3);
+  --denhaag-listitem-icon-hover-color: var(--denhaag-blue-3);
+
+  /* ListItemSecondaryAction */
+  --denhaag-listitem-secondaryaction-focus-color: var(--denhaag-blue-3);
+  --denhaag-listitem-secondaryaction-color: var(--denhaag-grey-4);
+  --denhaag-listitem-secondaryaction-right: 16px;
+  --denhaag-listitem-secondaryaction-padding: 4px;
+
+  /* ListItemText */
+  --denhaag-listitemtext-primarytext-focus-color: var(--denhaag-blue-3);
+  --denhaag-listitemtext-primarytext-hover-color: var(--denhaag-blue-3);
+  --denhaag-listitemtext-primary-color: var(--denhaag-grey-4);
+  --denhaag-listitemtext-primary-weight: 400;
+  --denhaag-listitemtext-secondary-color: var(--denhaag-grey-4);
+  --denhaag-listitemtext-secondary-weight: 400;
+  --denhaag-listitemtext-secondary-font-size: var(--denhaag-font-size-s);
+  --denhaag-listitemtext-secondary-primarytext-line-height: 32px;
```

Comment:
The naming here does get a bit complex, but i would prefer using the term `multi-line` instead of `secondary` here.

Reply:
It is necessary to specify secondary here since it is aimed at another element.
I agree this is confusing  since secondary was also used to specify primary text in the multiline case here.
So I renamed the secondary in the case of primarytext to multi-line and left all other secondary as they are.